service-worker.js

```diff
@@ -0,0 +1,33 @@
+---
+layout: null
+---
+const version = "{{ site.time | date:'%Y%m%d%H%M%S'}}";
+const cacheName = `cache-${version}`;
```

Comment:
Is `version` a string provided by jekyll? Under what circumstances will jekyll change that value? Any deployment?

Reply:
Yes, it will run on each new build. I realise that this is not optimal, but i dont have a better idea to invalidate the cache otherwise. 

Do you see this as a treat? Would it result in poor performance to force the app to have a new cache on each build?